api-tests/features/inherited-identity.feature

```diff
@@ -18,20 +18,110 @@ Feature: Inherited Identity
-    Then I get a 'PCL200' identity
-    When I start a new 'medium-confidence' journey
+    Then I get a '<expected-identity>' identity
+
+    When I start a new '<return-journey-type>' journey
     Then I get a 'page-ipv-identity-document-start' page response
 
-  Scenario: Migrate PCL 200 HMRC profile successfully with no evidence and returns with P2/PCL250
-    Given I start a new 'medium-confidence-pcl200-pcl250' journey with inherited identity 'alice-vot-pcl200-no-evidence'
+    Examples:
+    | inherited-identity           | expected-identity | return-journey-type      | vtr       |
+    | alice-vot-pcl200-no-evidence | PCL200            | medium-confidence        | P2        |
+    | alice-vot-pcl200-no-evidence | PCL200            | medium-confidence-pcl250 | P2/PCL250 |
+    | alice-vot-pcl250-no-evidence | PCL250            | medium-confidence        | P2        |
+
+  Scenario: Migrate PCL250 HRMC profile successfully with no evidence and returns with P2/PCL250
+    Given I start a new 'medium-confidence-pcl200-pcl250' journey with inherited identity 'alice-vot-pcl250-no-evidence'
     Then I get an OAuth response
     And an 'IPV_INHERITED_IDENTITY_VC_RECEIVED' audit event was recorded [local only]
     When I use the OAuth response to get my identity
-    Then I get a 'PCL200' identity
+    Then I get a 'PCL250' identity
     When I start a new 'medium-confidence-pcl250' journey
+    Then I get an OAuth response
+
+  Scenario Outline: P2 identity takes priority over successfully migrated PCL200 <evidence-state>
+    Given I start a new 'medium-confidence' journey
+    Then I get a 'page-ipv-identity-document-start' page response
+    When I submit an 'appTriage' event
+    Then I get a 'dcmaw' CRI response
+    When I submit 'kenneth-passport-valid' details to the CRI stub
+    Then I get a 'page-dcmaw-success' page response
+    When I submit a 'next' event
+    Then I get an 'address' CRI response
+    When I submit 'kenneth-current' details to the CRI stub
+    Then I get a 'fraud' CRI response
+    When I submit 'kenneth-score-2' details to the CRI stub
+    Then I get a 'page-ipv-success' page response
+    When I submit a 'next' event
+    Then I get an OAuth response
+    When I use the OAuth response to get my identity
+    Then I get a 'P2' identity
+
+    # New journey with inherited identity
+    Given I start a new 'medium-confidence-pcl200-pcl250' journey with inherited identity '<inherited-identity>'
+    Then I get a 'page-ipv-reuse' page response
+    When I submit an 'next' event
+    Then I get an OAuth response
+    And an 'IPV_INHERITED_IDENTITY_VC_RECEIVED' audit event was recorded [local only]
+    When I use the OAuth response to get my identity
+    Then I get a 'P2' identity
+
+    Examples:
+    | inherited-identity                | evidence-state   |
+    | alice-vot-pcl200-no-evidence      | with no evidence |
+    | kenneth-vot-pcl250-driving-permit | with evidence    |
```

Comment:
FWIW I think the level of confidence (PCL200/PCL250) is more relevant than the evidence state here